aws/resource_aws_securityhub_standards_control.go

```diff
@@ -0,0 +1,140 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"path"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/securityhub"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAwsSecurityHubStandardsControl() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAwsSecurityHubStandardsControlCreate,
+		ReadContext:   resourceAwsSecurityHubStandardsControlRead,
+		UpdateContext: resourceAwsSecurityHubStandardsControlUpdate,
+		DeleteContext: resourceAwsSecurityHubStandardsControlDelete,
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
```

Comment:
I suggest renaming this attribute to `standards_control_arn` to avoid ambiguity.

Reply:
Done